backend/dev.sh

```diff
@@ -3,5 +3,7 @@ set -eu
 pip -q install --upgrade pip
 pip -q install --cache-dir=.pip -r requirements.txt
 
+pip -q install --cache-dir=.pip -e /lib/Akvo_rabbitmq_client
```

Comment:
The URL for this can go into `requirements.txt` I think. If you try a `pip freeze` in the environment you'll see.